src/SafeAreaView.tsx

```diff
@@ -27,7 +29,20 @@ export const SafeAreaView = React.forwardRef<
           return accum;
         }, {})
       : edges;
-    return _edges ?? defaultEdges;
+
```

Comment:
Minor - but we convert the array to an object, then use the object to create a new object. We could

Reply:
(Not essential to change though)